vspace/src/main/java/edu/asu/diging/vspace/web/staff/StaffSpaceController.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.ExhibitionSpaceOrderMode;
+import edu.asu.diging.vspace.core.services.IExhibitionManager;
+import edu.asu.diging.vspace.core.services.ISpacesCustomOrderManager;
+
+/**
+ * StaffSpaceController is the controller
+ * to set mode of order of spaces on exhibition side, change custom order selection
+ * @author Tushar Anand
+ *
+ */
+
+@Controller
+public class StaffSpaceController {
```

Comment:
That's not a controller for spaces, it's for space ordering isn't it? Should be named more clearly.